traitsui/qt4/list_editor.py

```diff
@@ -382,6 +382,21 @@ def _mutable_default(self):
         """
         return self.factory.mutable
 
+    def _scrollable_default(self):
+        """ Trait handler to set the scrollable from the factory.
+        """
+        return self.factory.scrollable
+
+    def __list_pane_default(self):
```

Comment:
I'd just create this directly in the init; or if you want to encapsulate, have this as a private method which creates the `list_pane` attribute.